TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java

```diff
@@ -25,4 +26,13 @@ public void createNewProfile(String name, String type, String relatedData) {
         cbbRelatedData.select(relatedData);
         btnFinish.click();
     }
+
+    public void clickItemTypeComboBox() {
+        cbbItemType.click();
+    }
+
+    public boolean isOptionVisible(String itemType) {
```

Comment:
Select has an available method to get all the options.

Reply:
I fixed it. Please check again